src/locales/vocabulary.js

```diff
@@ -90,7 +91,7 @@ export default {
   'editdialog.cancel_button': 'Cancel',
   'editdialog.save_button_edit': 'Save to OSM',
   'editdialog.save_button_delete': 'Delete',
-  'editdialog.save_button_note': 'Add note',
+  'editdialog.save_button_note': 'Submit',
```

Comment:
I like to disinguish between "Add note" and "Save to OSM" on the button level. So this is more a UX choice. 

Unfortuntally the dialog looks almost the same, except for heading and the save button. Do you think it is ok to leave "Add note"?

I can imagine we can also design the dialog to look prominently different, eg. size, picture, bigger icon etc. that the button wouldnt be used to communicate the difference..

Reply:
Personally, I think at this point the user has already chosen to add a node, and is fully aware of what they are working on.
Putting "Add note" as the button to "submit"/"confirm" their edit or suggestion seems redundant and could be confusing.

This is a mostly irrelevant decision though, since it doesn't really matter what word is on that button, everyone will know what it does.

Reply:
I can see now, it might be confusing as a text. The button position is so prominent it is almost always clear. But I am ok with "Submit" for note now. 🙂